kpi/views/current_user.py

```diff
@@ -1,13 +1,128 @@
 # coding: utf-8
+from constance import config
 from django.contrib.auth.models import User
-from rest_framework import viewsets
+from django.db import transaction
+from django.utils.timezone import now
+from django.utils.translation import gettext as t
+from rest_framework import status, viewsets
+from rest_framework.response import Response
 
+from kobo.apps.trash_bin.utils import move_to_trash
 from kpi.serializers import CurrentUserSerializer
 
 
 class CurrentUserViewSet(viewsets.ModelViewSet):
+    """
+    <pre class="prettyprint">
+    <b>GET<b> /me/
+    </pre>
+
+    > Example
+    >
+    >       curl -X GET https://[kpi]/me/
+    >
+    >       {
+    >           "username": string,
+    >           "first_name": string,
+    >           "last_name": string,
+    >           "email": string,
+    >           "server_time": "YYYY-MM-DDTHH:MM:SSZ",
+    >           "date_joined": "YYYY-MM-DDTHH:MM:SSZ",
+    >           "projects_url": "https://[kpi]/{username}",
```

Comment:
it's actually a kobocat URL :grimacing: so `https://[kobocat]/{username}`
i think (& hope) this is used only for the "Projects (legacy)" button